runtime/server/queries_timeseries.go

```diff
@@ -165,18 +164,18 @@ func (s *Server) EstimateRollupInterval(ctx context.Context, request *runtimev1.
 	r := trr.TimeRangeSummary.Interval
 
 	const (
-		MICROS_SECOND = 1000 * 1000
-		MICROS_MINUTE = 1000 * 1000 * 60
-		MICROS_HOUR   = 1000 * 1000 * 60 * 60
-		MICROS_DAY    = 1000 * 1000 * 60 * 60 * 24
+		MicrosSecond = 1000 * 1000
+		MicrosMinute = 1000 * 1000 * 60
+		MicrosHour   = 1000 * 1000 * 60 * 60
+		MicrosDay    = 1000 * 1000 * 60 * 60 * 24
```

Comment:
Start with lowercase letter?

Reply:
can do either, mostly i have seen go docs using starting with capitals for constants - ref: https://go.dev/blog/constants#:~:text=of%20string%20constants.-,Floats,number%20with%20no%20imaginary%20part%2C%20that%20is%2C%20a%20real%3F%20Here%E2%80%99s%20one%3A,-const%20Two%20%3D%202.0

Reply:
Yeah, exported constants need to be capitalized, but these are not exported, so should just be lowercase :)